### PR TITLE
[FIX] INT-1011

### PR DESCRIPTION
**Pull request by tmeliotpg**
- Fixed INT-1011 
- Implemented wc_localize_script to init product simulator to increase code base readability and maintainability

### CODE REVIEW

File: templates/product_simulator.php [hunk 1]
```diff
@@ -65,6 +65,7 @@ function loadSimulatorPagantis()
     {
         if(typeof pgSDK == 'undefined')
         {
+            console.warn(typeof pgSDK)
```
Comment (pgarcess) [inline templates/product_simulator.php]: remove comments

File: templates/product_simulator.php [hunk 2]
```diff
@@ -104,9 +105,9 @@ function loadSimulatorPagantis()
             simulator_options.itemPromotedAmountSelector = priceSelector;
         }
 
-        if (typeof window.pgSDK != 'undefined') {
+        if (typeof window.pgSDK !== 'undefined') {
             window.WCSimulatorId = window.pgSDK.simulator.init(simulator_options);
-
+            console.trace();
```
Comment (pgarcess) [inline templates/product_simulator.php]: remove comments

File: assets/js/pg-product-simulator.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
+/** global console, simulatorData  */
```
Comment (pgarcess) [inline assets/js/pg-product-simulator.js:1]: If you add this file, you need to remove the code from the template

File: templates/product_simulator.php (hunk 1 above)
Comment (tmeliotpg) [reply templates/product_simulator.php]: The file is not used anymore, so I will remove the file completely

File: templates/product_simulator.php (hunk 2 above)
Comment (tmeliotpg) [reply templates/product_simulator.php]: The file is not used anymore, so I will remove the file completely

File: assets/js/pg-product-simulator.js (hunk 3 above)
Comment (tmeliotpg) [reply assets/js/pg-product-simulator.js:1]: I will remove the template product_simulator.php  because it's not used anymore

**Review by pgarcess — APPROVED** [review]

**Pull request merged**